qiskit/pulse/utils.py

```diff
@@ -41,3 +46,53 @@ def pad(schedule, channels=None, until=None):
     """
     warnings.warn("The function `pad` has been moved to qiskit.pulse.reschedule. It cannot be "
                   "invoked from `utils` anymore (this call returns None).")
+
+
+def measure(qubits: List[int],
+            schedule: Schedule,
+            inst_map: Optional[InstructionScheduleMap],
+            meas_map: List[List[int]],
+            backend: Optional['BaseBackend'] = None,
+            qubit_mem_slots: Optional[Dict[int, int]] = None) -> Schedule:
+    """
+    This is a utility function to measure qubits using OpenPulse.
+
+    Args:
+        qubits: List of qubits to be measured.
+        schedule: Schedule of the circuit.
+        inst_map: Mapping of circuit operations to pulse schedules. If None, defaults to the
+                  ``circuit_instruction_map`` of ``backend``.
+        meas_map: List of sets of qubits that must be measured together. If None, defaults to
+                  the ``meas_map`` of ``backend``.
+        backend: A backend instance, which contains hardware-specific data required for scheduling.
+        qubit_mem_slots: Mapping of measured qubit index to classical bit index.
+
+    Returns:
+        A schedule corresponding to the inputs provided.
+    """
+
+    inst_map = inst_map or backend.defaults().circuit_instruction_map
+    meas_map = meas_map or backend.configuration().meas_map
```

Comment:
I think these two lines need to be in a try/except block. For instance, if `inst_map` and `backend` are both `None`, this will raise an AttributeError